pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -477,9 +477,8 @@ void processArguments() throws Exception {
 
         protected void validateFunctionConfigs(FunctionConfig functionConfig) {
 
-            if (isBlank(functionConfig.getOutput()) && !functionConfig.isSkipOutput()) {
-                throw new ParameterException(
```

Comment:
I thought with #2261 we decided with @sijie 's feedback to "fail fast" if there is a conflict because output-topic name and skip-output are mutually exclusive and user should not use it together. so, does it make sense to give error-message to clarify role of the input skipoutput flag.

Reply:
didn't we agree on #2261 to fail-fast if no output specified and if user want to skip output, they should skip-output?